### ISSUE TITLE
Drop this lesson and rely on Carpentries Shell lesson

### ISSUE THREAD

**Issue by ocaisa** [hpc-carpentry/hpc-shell#45]
We've discussed in a recent coordination meeting dropping the HPC shell and instead having the Carpentries shell lesson as a prerequisite for the HPC intro lesson. Looking quickly at the lesson, there is really only one episodes in the lesson that should be fully integrated into the HPC intro (and even this already has some overlap):

- [Connecting to a remote HPC system](http://www.hpc-carpentry.org/hpc-shell/01-connecting/index.html)

There may be some other nice points but the majority of things are actually already touched in HPC intro.

**Comment by ocaisa** [comment]
One thing about adding that section is that (at least in my experience) JupyterHub interfaces are becoming increasingly common and are very powerful. It might be worth adding this option for getting a remote shell.

**Comment by bkmgit** [comment]
They are becoming common.  Lesson should accommodate this, but not require use of Jupyter as it is also resource intensive, ssh works well in a wide variety of situations, though takes more time to setup.

**Comment by ocaisa** [comment]
Yes, totally agree, I mean it as an alternative connection option. If we bring that episode into HPC intro, I think we could extract part of it ("Logging onto the system") as a snippet , the actual mechanism for connecting is likely to vary a lot site-to-site and a snippet might be the easiest way for people to integrate their custom information.
